plugins/amazonq/chat/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevControllerTest.kt

```diff
@@ -390,4 +394,119 @@ class FeatureDevControllerTest : FeatureDevTestBase() {
         newFileContentsCopy[0].rejected = !newFileContentsCopy[0].rejected
         coVerify { messenger.updateFileComponent(testTabId, newFileContentsCopy, deletedFiles, "") }
     }
+
+    @Test
+    fun `test modifyDefaultSourceFolder customer does not select a folder`() = runTest {
```

Comment:
Why are we adding the runTest call here specifically?

Reply:
Because the test is verifying and running coroutines and need to be under a scope, it feels cleaner to add the `runTest` enclosing the whole test.